string AddressToFind    # red/yellow/blue/orange/BuildingPlace
---
bool flag